src/NuGet.Clients/NuGet.CommandLine/NuGetCommand.resx

```diff
@@ -351,6 +351,33 @@ nuget push foo.nupkg -Timeout 360</value>
   <data name="PushCommandUsageSummary" xml:space="preserve">
     <value>&lt;package path&gt; [API key] [options]</value>
   </data>
+  <data name="SearchCommandDescription" xml:space="preserve">
+    <value>Searches a given source using the query string provided. If no sources are specified, all sources defined in %AppData%\NuGet\NuGet.config are used.</value>
+    <comment>Please don't localize "%AppData%\NuGet\NuGet.config", "NuGet.config".</comment>
+  </data>
+  <data name="SearchCommandPreRelease" xml:space="preserve">
+    <value>Include prerelease packages.</value>
+  </data>
+  <data name="SearchCommandSourceDescription" xml:space="preserve">
+    <value>A list of package sources to search.</value>
```

Comment:
Can/should we mention the separator in between sources?

Reply:
I'm not sure what you mean. Are you referring to the way the results from multiple sources are displayed? Or is it something about how you pass multiple sources as optional arguments (`-Source 'source1' -Source 'source2'`)?

Reply:
We chatted offline. I didn't realize you could provide the `-Source` option multiple times. I suggested changing the description to something like:

```
The package source to search. You can pass multiple -Source options to search multiple package sources
```

Reply:
I've gone ahead and changed the description based on your suggestion.